models/transformer.py

```diff
@@ -144,13 +144,16 @@ def __init__(self, feature_name_to_idx, embedding_kwargs, use_masked_mha, num_la
     def call(self, x, mask, training):
         x = self.feature_embedding(x)
         x = self.dropout(x, training=training)
+        
+        attn_scores = []
```

Comment:
Do you know how the attention score aggregation adds up to the training/inference time? I would expect it doesn't but just in case.